src/Snackbar/Snackbar.js

```diff
@@ -0,0 +1,247 @@
+// @flow weak
+
+import React, { PropTypes, Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import SnackbarContent from './SnackbarContent';
+import ClickAwayListener from '../internal/ClickAwayListener';
+
+
+export const styleSheet = createStyleSheet('MuiSnackbar', (theme) => {
+  const { zIndex, transitions, breakpoints } = theme;
+
+  const transitionString = `${transitions.create('transform', {
```

Comment:
You are using the same animation options, please use an array for `transform` and `visibility` as first argument